jdi-light/src/test/java/io/github/epam/weblisttests/WebListTests.java

```diff
@@ -0,0 +1,57 @@
+package io.github.epam.weblisttests;
+
+import com.epam.jdi.light.elements.complex.WebList;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import static org.testng.Assert.assertEquals;
+
+public class WebListTests {
+    private String element0Text = "0 element";
+    private String element1Text = "1 element";
+    private By locator = new By.ByCssSelector("");
+    private List<WebElement> list;
+    private WebList webList;
+    private WebElement element0;
+    private WebElement element1;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'element1' could be replaced by a local variable.](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=4816272)